test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -689,7 +689,8 @@ static odp_crypto_session_t session_create(odp_crypto_op_t op,
 	 * In some cases an individual algorithm cannot be used alone,
 	 * i.e. with the null cipher/auth algorithm.
 	 */
```

Comment:
There are two fixes in this PR (they could have been in two different patches as well).

The first fix is the way ODP_CRYPTO_SES_ERR_ALG_COMBO is checked. That does not fix b7d520d but c14955e1d (validation: crypto: do not test unsupported algorithm combos).

The second fix fixes the hash result overlap test in case an auth algorithm is not supported with the null cipher algorithm. That fixes b7d520d, but the commit message does not otherwise mention the fix.

The commit messsage should be updated to mention both issues in the description, or better yet, you could split the patch to two. Also, there should be a fixes-line for each of the original patches or none at all. The CC-line could be removed.

Reply:
Ack. Separated the two fixes into two different commits with v3.